Automattic/wp-calypso
# Remove wpcom.undocumented().mailingList() methods

Remove the `wpcom.undocumented().mailingList()` helper from the global `wpcom` object and instead issue the REST requests directly at call sites with `wpcom.req.post`. Should remove code from the entrypoint chunk.

**How to test:**
Find a wordpress.com newsletter email in your inbox, if you have any, and use the `/mailing-lists/unsubscribe` link there, only updated for `calypso.localhost` or `calypso.live`. Verify that the REST requests to unsubscribe and resubscribe you are issued and are successful.
